src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConstants.java

```diff
@@ -20,4 +20,6 @@
   public static final String TABLE_NAME_KEY = "table";
   public static final String QUERY_NAME_KEY = "query";
   public static final String QUERY_NAME_VALUE = "query";
+  public static final String OFFSET_PROTOCOL_VERSION_KEY = "version";
```

Comment:
I wonder if this should be "protocol" rather than "version", since the latter might be mistaken as the Connect version.